code/graphics/opengl/gropengltnl.cpp

```diff
@@ -308,8 +308,9 @@ void* gr_opengl_map_buffer(gr_buffer_handle handle)
 	Assertion(GLAD_GL_ARB_buffer_storage != 0, "Persistent mapping is not available in this OpenGL context!");
 
 	opengl_bind_buffer_object(handle);
+	glBufferData(buffer_obj.type, buffer_obj.size, nullptr, buffer_obj.gl_usage);
```

Comment:
This call is wrong. In the case of persistent mapping we need to specify the `GL_MAP_PERSISTENT_BIT` flag via  `glBufferStorage`. Since this has already happened before this call, this is unnecessary anyway.